cms/djangoapps/contentstore/views/tests/test_textbooks.py

```diff
@@ -19,7 +19,7 @@ def test_view_index(self):
         self.assertEqual(resp.status_code, 200)
         # we don't have resp.context right now,
         # due to bugs in our testing harness :(
-        if resp.context:
+        if resp.context and resp.context.get('course'):
```

Comment:
Rendering `commons.js` within this Django template makes `context` no longer empty, but `'course'` is still not populated. I don't know what the aforementioned 4-year-old "bugs in our testing harness" are about, but they're out of the scope of this PR.